### ISSUE TITLE
Global settings .RStudioProfile file?

### ISSUE THREAD

**Issue by kendonB** [rstudio/rstudio#5301]
I wasn't able to find an existing request for this but apologies if there is one already.

I am frustrated every time I reinstall RStudio on a new machine that I have to change the settings manually. It would be nice to just have a settings file that I can export and save.

The old site has mentioned it (https://support.rstudio.com/hc/en-us/community/posts/115000932128-RStudio-Config-Files, https://community.rstudio.com/t/rstudio-configuration-files/19750/3) but I didn't see it here as a feature for people to thumbs up.

**Comment by Robinlovelace** [comment]
Quick question on this @jmcphers, how would one change the settings for all users? I assume that 

```
~/.config/rstudio/rstudio-prefs.json
```

only works on a per user basis.

To provide a concrete example, I'm looking for a way to turn off in-line chunk output in .Rmd files as per https://github.com/Robinlovelace/geocompr/issues/443.

I had a good read of https://github.com/rstudio/rstudio/pull/5069, as you suggested, and the solution still alludes me.

**Comment by jmcphers** [comment]
The settings file for all users is located here by default:

    /etc/rstudio/rstudio-prefs.json

Does that help?

**Comment by Robinlovelace** [comment]
It does indeed 👍 

Sorry for follow-up question, but how would I modify that file to prevent in-line code chunk output?

**Comment by jmcphers** [comment]
Try this:

```
{
    "rmd_chunk_output_inline": false
}
```

**Comment by Robinlovelace** [comment]
Thanks for the info @jmcphers, tried it on RStudio Versions 1.2.5001 and 1.2.1578 but to no avail. I tried adding that text to `/etc/rstudio/rstudio-prefs.json` and restarting the server but code output was still inline. Do I need to wait for a more recent version?

**Comment by jmcphers** [comment]
Yes. This feature is only in the development (1.3) branch of RStudio. You can get a daily build if you want to try it:

https://dailies.rstudio.com/

**Comment by Robinlovelace** [comment]
Confirmed, it worked in rocker's daily images. Many thanks!

**Comment by jmcphers** [comment]
Great, glad to hear it's finally working for you!

**Comment by pat-s** [comment]
@jmcphers Sorry for bumping again. Could you tell me the location of the global settings file on Windows? I want to change the defaults for multiple users on a shared machine. Gave up after 15 min of search :/

**Comment by jmcphers** [comment]
It's in `%PROGRAMDATA%\RStudio`. Documentation is here:

https://docs.rstudio.com/ide/desktop-pro/latest/settings.html#preferences

**Comment by Robinlovelace** [comment]
In case it's of use/interested I've just succeeded in creating a Docker image with modified RStudio settings: https://github.com/geocompr/docker/commit/d14dc4aa9a091b10b1e22ef0d16316f9a50750fb

**Comment by benwhalley** [comment]
Can I confirm where the global prefs file is supposed to be? It doesn't seem to be picking changes up for me:

```
ben@rstudio:~$ cat /etc/rstudio/rstudio-prefs.json 
{
    "save_workspace": "never",
    "check_arguments_to_r_function_calls": true,
    "check_unexpected_assignment_in_function_call": true,
    "warn_if_no_such_variable_in_scope": true,
    "warn_variable_defined_but_not_used": true,
    "style_diagnostics": true,
}
```

Also - are these prefs applied to users when accounts are created, or merged with any other local settings a user has made through the GUI?
